# This message contains a crop description
# (0, 0) is at top-left corner of image
#
float64 row_orientation         # crop rows orientation angle with respect to image frame
#float64 area
#float64 height
#float64 width
#geometry_msgs/Point[] contour    # contour contains the vertices coordinates of crop contour
crop_image_processing/Image_point[] contour
